features/20-exchange-access-token.feature

```diff
@@ -0,0 +1,61 @@
+@smart
+Feature: App exchanges authorization code for access token
+
+    Scenario: Success response has all required parameters
+        Given OAuth is enabled
+        And I am not logged in
+        When I ask for authorization
+        And I exchange my grant code
+        Then the response code should be 200
+        And the JSON response will contain access_token
+        And the JSON response will contain token_type
+        And the JSON response will contain scope
+        And the JSON response will contain state
```

Comment:
The official spec for the access_token response is at https://tools.ietf.org/html/rfc6749#section-4.2.2 -- which says:

    state
          REQUIRED if the "state" parameter was present in the client
          authorization request.  The exact value received from the
          client.

You're saying nobody's including this value? We all should be!